src/content/docs/browser/browser-monitoring/browser-pro-features/session-replay/troubleshooting-session-replay.mdx

```diff
@@ -0,0 +1,77 @@
+---
+title: "Troubleshooting session replay"
+metaDescription: "Troubleshoot common issues with session replay in New Relic Browser."
+freshnessValidatedDate: 2024-12-19
+---
+
+This is the first step in diagnosing your New Relic Session replays problem yourself, use this guide to resolve issues quickly.
```

Comment:
the second half of this sentence adds no value. consider removing.

Reply:
Updated